tools/gcf_init/deploy_gcf.py

```diff
@@ -11,18 +11,20 @@
 if len(sys.argv) > 1:
   function_names = [sys.argv[1]]
 else:
-  function_names = ['gettasks', 'getrecenttasks']
+  function_names = ['gettasks', 'getrecenttasks', 'closetask', 'closetasks']
 
 config.LoadConfig()
 
 for function in function_names:
   print 'Deploying function {0:s}'.format(function)
   cmd = ('gcloud --project {0:s} beta functions deploy {1:s} --stage-bucket '
          '{2:s} --region {3:s} --trigger-http'.format(config.PROJECT, function,
-                                       config.BUCKET_NAME, config.TURBINIA_REGION))
+                                                      config.BUCKET_NAME,
```

Comment:
Yeah, it could probably be moved to a better location to make it more obvious, but it's here:
https://github.com/google/turbinia/blob/master/turbinia/config/style.yapf

Don't worry about reformatting for this PR though; I need to make some changes to the config and re-run it, and so I'll have to re-yapf everything then.